epub34/annotations/index.html

```diff
@@ -31,52 +31,84 @@
-			<p> W3C Annotations have 0 or more Bodies, whereas this document only defines a single Body
-				per annotation. Such Body can be remote or embedded, whereas this document defines embedded Bodies only 
-				for textual comments (the inclusion mechanism is to be defined for audiovisual comments). </p> 
-			<p> W3C Annotations have 1 or more Targets, whereas this document only defines a single Target
-				per annotation. The Target of an EPUB Annotation is the content being annotated, which is  
-				a specific segment in a document within the EPUB package, defined by its relative Source URL and a Selector. </p>
-			<p> W3C Annotations can have multiple motivations, whereas this document only defines a single motivation
-				per annotation. </p>
-			<p> W3C Annotations can have multiple creators, whereas this document only defines a single creator
-				per annotation. </p>
-			<p> W3C Annotations can have additional properties not defined in this document, like a specific generator application
-				and generated date per Annotation, whereas this document only defines these properties at the Annotation Set level. 
-				W3C Annotations also handle more property values and types than defined in this document. </p>
-			<p> W3C Annotations define an AnnotationCollection structure to handle paginated requests, whereas this document
-				defines an AnnotationSet structure to group multiple annotations for sharing purposes.
-				Implementers MUST support AnnotationSet and MAY support both structures for maximum compatibility. </p>
-			<p> Note: This document does not define how annotations are created, stored, or
+        <section id="conformance">
+            <!-- <p>All algorithm explanations are <em>non-normative</em>.</p> -->
+        </section>
+
+		<section id="1-profile-of-the-w3c-annotation-data-model" class="informative">
+			<h1> Subset of the Web Annotation Data Model </h1>
+			<p> This section defines a profile of the [[[annotation-model]]] [[annotation-model]], as used for EPUB
+				Annotations.</p>
+            <p> In the Web Annotation model, the core structure is the
+                <a data-cite="annotation-model#annotations">Annotation object</a>,
+                which contains properties defining the annotation's
+                <a data-cite="annotation-model#bodies-and-targets">Body and Target</a>.
+                EPUB Annotations reuse the <a href="#1-1-annotation-object">same model</a> with some restrictions listed below.
+                Subsequent sections provide more formal definitions for the terms used by this
+                specification.
+            </p>
+           <ul>
+                 <li> Web Annotations may have 0 or more Bodies, whereas this document requires to have a single <a href="#1-4-body">Body</a>
+                    per annotation. Furthermore, in the [[annotation-model]], such Body can be remote or embedded,
+                    whereas only embedded Bodies are used in EPUB Annotation
```

Comment:
We could put the focus on EPUB in this section by defining the EPUB Annotation requirement first, and then noting the variation from Web Annotations. 
```suggestion
                    EPUB Annotations must have only a single <a href="#1-4-body">Body</a>
                    per annotation. Only embedded Bodies are used in EPUB Annotation for textural comments. Unlike the Web annotation [[annotation-model]], which may have 0 or more Bodies, and where such Body or Bodies can be remote or embedded.
```

Reply:
I am o.k. doing that. @llemeurfr ?

Reply:
Stylistically, it may be better to combine this with the next sentence: having those two mentions of "EPUB Annotation" makes it a bit heavy for my ears. @sueneu, what do you think?

Reply:
I agree with @Ivan that the suggestion makes sense only if lines 77-80 are replaced. Here, only line 79 is replaced by two lines, it does not seem to be what @sueneu was willing to do. 

I'm ok to use an active voice, something like "EPUB Annotations have a single <a href="#1-4-body">Body</a>, whereas W3C Annotations support 0 or more Bodies."

Reply:
It would be fine with me to replace 77-80.

People reading the spec are most interested in what is required for EPUB Annotations. With that in mind, we should start the paragraph with those requirements, and write them as clearly as possible. Next, add the divergence from the web annotations. 

 @llemeurfr suggested active voice would be a good option for clarity.

Reply:
This comment was meant to float an idea about the entire section, not as a specific word-for-word change.
Apologies if this was done in a clumsy way.

For clarity/efficiency, we can delete this comment before merging the PR. Then we can edit the next version to reflect a more active voice and a focus on EPUB Annotations first.

Reply:
> This comment was meant to float an idea about the entire section, not as a specific word-for-word change. 
> 
> For clarity/efficiency, we can delete this comment before merging the PR. Then we can edit the next version to reflect a more active voice and a focus on EPUB Annotations first.

Let us do that. I will merge this PR now, and we should look at it further linguistically